src/main/java/mate/academy/hibernate/relations/dao/impl/CountryDaoImpl.java

```diff
@@ -2,8 +2,11 @@
 
 import java.util.Optional;
 import mate.academy.hibernate.relations.dao.CountryDao;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
```

Comment:
Ensure that the `DataProcessingException` class is correctly implemented and available in the specified package. This exception is crucial for handling data processing errors.